web-admin/src/features/projects/user-invite/UserInviteForm.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+  import { createAdminServiceAddProjectMember } from "@rilldata/web-admin/client";
+  import UserRoleSelect from "@rilldata/web-admin/features/projects/user-invite/UserRoleSelect.svelte";
+  import { Button } from "@rilldata/web-common/components/button";
+  import MultiInput from "@rilldata/web-common/components/forms/MultiInput.svelte";
+  import { createForm } from "svelte-forms-lib";
```

Comment:
Now that @briangregoryholmes added [SuperForms](https://superforms.rocks/), we should migrate off `svelte-forms-lib` toward `sveltekit-superforms`. It can be a gradual migration. Currently, `RenameAssetModal` and `CreateShareableURLForm` both use SuperForms and can be inspiration.

Reply:
Updated to use SuperForms